.github/workflows/javascript-tests.yml

```diff
@@ -0,0 +1,21 @@
+name: Continuous testing
+
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use node 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Run tests
+        run: yarn test
+      - name: Run lint
+        run: yarn lint
+      - name: Build the project
+        run: yarn build
```

Comment:
But lint and test are already in gitlab pipeline, why not add build there?

Reply:
Oh that's true. Removing it then

Reply:
Or would it make more sense to run the testing in GitHub actions and the deployment in GitLab?

Reply:
Since the deployment is already in Gitlab I think it makes sense to have all CI jobs there